tests/nested/manual/fde-on-classic/task.yaml

```
@@ -126,6 +126,7 @@ execute: |
   # Test aborting a kernel installation before a reboot
   refresh_rebooting_snap pc-kernel-new.snap pc-kernel no-reboot
   remote.exec sudo snap abort --last=install
+  remote.exec sudo snap watch --last=install
```

just checked that snap watch exits with a error if the change was canceled (and possibly on failure too), so this should likely be:

```suggestion
  remote.exec sudo snap watch --last=install || true
```

---

ok, let's try again, thanks